spec/rails_helper.rb

```diff
@@ -133,9 +137,9 @@ def with_translation(translation)
 
 # All RSpec configuration needs to happen before any examples
 # or else it whines.
-require 'integration_example_group'
+require 'support/active_admin_request_helpers'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.